Java/Interfaces/Static_Methods_In_Interfaces/Interface_Static Methods.md

```diff
@@ -0,0 +1,56 @@
+# Interfaces: A Recapitulation
+
+- As you may already be knowing from the previous modules, interfaces are basically blueprints of a class, which can possess abstract methods having no *concrete* method body, along with variables. 
+
+- The main objective is to gain the property of abstraction, which is essentially hiding the sophisticated details of an object's implementation and exposing only those which may be deemed necessary, in OOPS parlance.
+
+- Another property that can be availed is that of multiple inheritance (Java doesn't allow direct implementation of it through ***extends*** keyword of classes).
+
+---
+
+### Sample Interface:
+
+
+```
+interface Music {
+
+ 
+  double vol, bass;
+ public abstract void increaseVolume();
+ void adjustBass(); //every method declared inside is public and abstract by default
+ 
+ }
+ 
+ public class Party implements Music {
+ 
+ public static void main (String [] args)
+ {
+   public abstract void increaseVolume()
+     {
+        System.out.println("Increased!!");
+       }
+   void adjustBass()
+   {
+      System.out.println("Adjusted!!");
+    }
+  }
+ }
+ 
+ ```
+ ---
+ 
+ # Static Methods In Interfaces:
+ 
+ Before we delve deep into static methods used in interfaces, let us first fully comprehend what a static method actually is.
```

Comment:
It seems incomplete, is it?

Reply:
@AkMo3 I created these commits (23 hours ago ones) for a different issue (issue #2701  to be precise) and haven't created a PR for the same. I don't know how this got mixed up with this PR pertaining to issue #1615.

Reply:
@AkMo3 I think since the previous PR wasn't merged on time, whatever new changes for a different issue I made have got reflected in this PR (since I'm working on the same branch). So, if you could allow me some time, I'll complete the documentation part for that issue so that you could review and finally merge both the documents. Is that okay?

Reply:
I will fix that no worries